src/epochs.lua

```diff
@@ -1,9 +1,8 @@
-local gar = require("gar")
-local crypto = require("crypto.init")
-local utils = require("utils")
-local balances = require("balances")
-local arns = require("arns")
-local constants = require("constants")
+local gar = require(".src.gar")
+local crypto = require(".crypto.init")
```

Comment:
Similar to other comment: should this be `.src.crypto.init`?

Reply:
https://github.com/ar-io/ar-io-network-process/pull/384#discussion_r1960534138